v2/passwordless/serverless/with-aws-lambda/setup-api-gateway.mdx

```diff
@@ -0,0 +1,64 @@
+---
+id: setup-api-gateway
+title: Set up API Gateway
+hide_title: true
+---
+
+<!-- COPY SECTION -->
+<!-- ./thirdpartyemailpassword/serverless/with-aws-lambda/setup-api-gateway.mdx -->
+<!-- 1 -->
+
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs";
+import TabItem from '@theme/TabItem';
+
+# Set up API Gateway
+
+## 1) Create a REST API Gateway
+
+We will be using AWS API Gateway to create a REST API that will be used to communicate with our Lambda functions. 
+
+<img src="/img/integration-lambda/create-api-gateway.png"/>
+
+## 2) Set up auth routes
+
+Create a `/auth` resource and then `/auth/{proxy+}` resources. This will act as a catch-all for all supertokens auth routes. **Enable CORS** while creating the proxy resource. 
+
+<img src="/img/integration-lambda/create-proxy-route.png"/>
+
+<img src="/img/integration-lambda/route-creation-complete.png"/>
+
+## 3) Attach lambda to the `ANY` method of the proxy resource
+
+Click on the "ANY" method and then "Integration" to configure the lambda function. Check **Lambda proxy integration** and then select your lambda function.
+
+<img src="/img/integration-lambda/configure-lambda-integration.png"/>
+
+## 4) Enable CORS for the proxy path
+
+Click on the `{proxy+}` resource and then "Enable CORS" button to open the CORS configuration page. 
+
+<img src="/img/integration-lambda/click-enable-cors.png" />
+
+<AppInfoForm
+    askForWebsiteDomain
+    hideWebsiteBasePathField
+>
+
+- In the CORS configuration page do the following:
+    - Add `rid,fdi-version,anti-csrf,st-auth-mode` to the existing `Access-Control-Allow-Headers`
+    - Set `Access-Control-Allow-Origin` to `'^{form_websiteDomain}'`
+    - Set `Access-Control-Allow-Credentials` to `'true'`. **Don't miss out on those quotes else it won't get configured correctly.**
+
+<img src="/img/integration-lambda/configure-cors.png"/>
+
+</AppInfoForm>
+
+## 4) Deploy the API Gateway
+
+Click the **Deploy API** button in the top right corner to deploy the API. During deployment, you'll be prompted to create a stage; for this tutorial, name the stage `dev`. After deployment, you will receive your `Invoke URL`.
+
+:::note
+Update `apiBasePath` and `apiGatewayPath` in the Lambda configuration if they have changed post API Gateway configuration.
+:::
```

Comment:
- you should also mention that you need to update the frontend and backend apiDomain value to be the domain that is provided to you by aws once you deploy.
- Change the `:::note` to `:::important`